src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -38,11 +37,8 @@ public void Start(DynamoModel model)
             if(service.GetTrackerFactory(GATrackerFactory.Name) == null)
                 service.Register(new GATrackerFactory(ANALYTICS_PROPERTY));
 
-            if (true == Configuration.DebugModes.IsEnabled("ADPAnalyticsTracker"))
```

Comment:
Any reason we are bringing this out of debug modes on master?

Reply:
Yes. With this PR I plan to control the optin/optout values from the UI.
We can register the ADP tracker here.even when the debugmode is not enabled. It is a step forward to fully enabling ADP.
If the adp debugmode is off, no analytics will pass through because of the new filtering mechanism (line 191).